tools/sgartifactregistry/auth.go

```diff
@@ -0,0 +1,53 @@
+package sgartifactregistry
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/tools/sggcloud"
+)
+
+// Develop starts the Cloud Run service at the provided Go path with the provided service account and config.
+func NPMAuthenticate(ctx context.Context, registryURL string) error {
```

Comment:
Doesn't matter much, but looks slightly better IMO 🤠 

```suggestion
func NpmAuthenticate(ctx context.Context, registryUrl string) error {
```

Reply:
Go is a little specific on abbreviations so URL is more common (and the linter will scream about it otherwise). I can change the npm one.

Reply:
Ah I see, go ahead with what's more common 👍